README.md

```diff
@@ -29,3 +29,12 @@ To exit the server, type Ctrl-C.
 ## Development
 
 Most of the interesting stuff is in the `routes` folder, particularly `drivers.js`.
+
+## Triggering Server to send fake WebSocket events for multiple vehicle demo
+
+With the server running in another terminal window:
+
+    curl -X GET http://localhost:3200/multiple_vehicles/send_events
```

Comment:
I would make this a `POST` (since it is doing something rather than retrieving something) and name the route something like `/simulation/start`.

If this is most often going to be started via the command line, I would also make an npm script in package.json so that you can run it with something like `npm simulate` (which would make the HTTP request above). Not really necessary, just a little nicety.

Other than that, looks good and makes sense.